prerequisites/software.rst

```diff
@@ -69,18 +70,26 @@ The following reverse proxies are supported:
 5. Elasticsearch (optional)
 ===========================
 
-.. note:: Package install will insist on installing elasticsearch, you can break dependencies during install if needed.
+For excellent search performance we use Elasticsearch. 
+While package install will insist on installing Elasticsearch, you can break those dependencies if needed.
```

Comment:
I'm not sure I fully understand this part. Is the rewording below correct?

```suggestion
Zammad uses Elasticsearch to improve the performance of its search bar.

If you wish, you can safely override the Elasticsearch dependency requirement
in Zammad’s package installer, but be warned that search performance will suffer.
```

Reply:
> Zammad itself doesn't have a package manager, so technically you'd have to force the package installer of your system (e.g. zypper, yum or apt)...

Yes, this is what I meant, but I see now that the way I wrote it was confusing.

```suggestion
Zammad uses Elasticsearch to
1) make search faster and
2) support advanced features like reports
or searching by email attachment contents.
This becomes increasingly important
as the number of tickets in your system gets larger and larger.

This dependency is optional but strongly recommended;
Zammad will work without it,
but search performance will be degraded, and some features will be disabled.

.. hint:: 📦 **If you install Zammad via package manager...**

   It’s perfectly safe to manually override the Elasticsearch dependency.
   The appropriate command line flag will depend on your platform
   (*e.g.,* ``--force``, ``--ignore-depends``, ``--skip-broken``);
   check your package manager’s manpage to find out.
```

I think that with the second paragraph above, it's safe to remove the "warning" directive immediately that follows this segment.